caluma/caluma_analytics/schema.py

```diff
@@ -132,6 +145,16 @@ class Meta:
         connection_class = CountableConnectionBase
 
 
+class AggregateField(DjangoObjectType):
```

Comment:
explicitly define the Enum name for `AggregateField` as `Function` to display choices as enum keys similar to `StartingObject` above and hook it up with the serializer, too